jersey-servers/src/test/resources/test-server.yml

```diff
@@ -6,6 +6,8 @@ server:
   adminConnectors:
     - type: http
       port: 0
+  gzip:
```

Comment:
why this change?

Reply:
Dropwziards builtin gzip filter gets in the way of remoting server compression filter. I think they should do the same thing but dropwizard returns different vary headers

Reply:
this is only for CompressionFilterTest, ja?

Reply:
yes, just for those tests